__tests__/components/editor/ResourceTemplateForm.test.js

```diff
@@ -13,7 +13,7 @@ describe('<ResourceTemplateForm /> functional testing', () => {
     rtId={'resource:schema:Name'}
     resourceTemplate={ basicRt } />)
 
- const testId = jest.spyOn(shortid, 'generate').mockReturnValue('abcd45')
+ jest.spyOn(shortid, 'generate').mockReturnValue('abcd45')
```

Comment:
If I understand correctly, you are using jest.spyOn() here not as a spy but as a stub. Given that, I wonder whether you would consider using jest.mock() instead.

Reply:
I couldn't find a way to make it work with just `jest.mock()` but the unused const testId was bothering me. I found a different solution which is:
```
const shortid = require('shortid')
 shortid.generate = jest.fn().mockReturnValue('abcd45')
``` 
Do you think this is better?

Reply:
@jgreben Yep! That's even better. How would you feel about using `import ... from` instead of `const ... require()`? It looks like we do use both in the editor. Looks like we use import nearly 400 times and we use require 45 times, so we're inching closer to consistency here. This is minor, admittedly.